unlock-app/src/components/content/certification/CertificationDetails.tsx

```diff
@@ -249,7 +254,11 @@ export const CertificationDetails = ({
       } else {
         return (
           <span>
-            You are viewing the certificate issued to the recipient.{' '}
+            You are viewing the certificate issued to{' '}
+            <span className="font-semibold ">{`${minifyAddress(
```

Comment:
Please link to the block explorer here, especially if you truncate the address.

Reply:
<img width="600" alt="Screenshot 2023-04-11 at 21 04 51" src="https://user-images.githubusercontent.com/20865711/231263892-fcb00699-3fb8-436f-bf8f-0f9d9812915c.png">